libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -98,6 +98,34 @@ void AP_Airspeed_MS4525::_measure()
     }
 }
 
+/*
+  this equation is an inversion of the equation in the
+  pressure transfer function figure on page 4 of the datasheet
+  
+  We negate the result so that positive differential pressures
+  are generated when the bottom port is used as the static
+  port on the pitot and top port is used as the dynamic port
+*/
```

Comment:
A bit nitpicking but this comment should really be before the *diff_press_PSI* variable since that's what it applies to